chainercv/utils/image/read_image.py

```diff
@@ -22,6 +22,11 @@ def _read_image_cv2(path, dtype, color):
         # reshape (H, W) -> (1, H, W)
         return img[np.newaxis].astype(dtype)
     else:
+        # alpha channel is inclued
+        if img.shape[-1] == 4:
+            bgr = img[:, :, :3]
+            alpha = img[:, :, 3:] / 255
+            img = bgr * alpha + 255 * np.ones_like(bgr) * (1 - alpha)
```

Comment:
I think there are three options.
1. just ignore alpha channel
2. blend with black image
3. blend with white image

Why do you choose 3.? Personally, I prefer 1.

Reply:
This is because the alpha channel contains meaningful information (for some part, the value is in `(0, 1)`).

Reply:
2. is not acceptable?

Reply:
Is 2 better?
I thought 3 was more natural